app/src/main/java/com/concordium/wallet/ui/account/accountsoverview/AccountsOverviewViewModel.kt

```diff
@@ -212,12 +232,12 @@ class AccountsOverviewViewModel(application: Application) : AndroidViewModel(app
     }
 
     fun initiateFrequentUpdater() {
```

Comment:
Please, remove `Frequent` from the name, it assumes that there are 2 different updaters while we have only one